### PR TITLE
Convert to es6 format.

### PR DESCRIPTION
**Pull request by fluffels**

### CODE REVIEW

File: js/jquery.plugincreator.es6 [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+ */
+/**
+ * TODO: Add checks to prevent over-writing plugins
+ * TODO: Add ability to not specify plugin name and have a randomly generated name assigned
+ */
+"use strict";
+
+import esprima from "esprima";
+
+var noOp = jQuery.noop,
```
Comment (OOPMan) [inline js/jquery.plugincreator.es6]: @fluffels Rather use `let` here as all of the items in this `var` list are intended to be constants...

File: js/jquery.plugincreator.es6 [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+ * Copyright 2014 Adam Jorgensen
+ * Released under the MIT license.
+ * http://github.com/OOPMan/jquery-plugin-creator/LICENSE
+ *
+ */
+/**
+ * TODO: Add checks to prevent over-writing plugins
+ * TODO: Add ability to not specify plugin name and have a randomly generated name assigned
+ */
+"use strict";
```
Comment (OOPMan) [inline js/jquery.plugincreator.es6]: @fluffels I believe this is no longer required for ES6 as it is now always strict.

**Comment by OOPMan** [comment]
@fluffels I take it I will need to push this out to NPM when the merge is done, right?

**Comment by fluffels** [comment]
@OOPMan nope :) npm can pull direcntly from GitHub

File: js/jquery.plugincreator.es6 (hunk 2 above)
Comment (fluffels) [inline js/jquery.plugincreator.es6]: @OOPMan http://stackoverflow.com/questions/31685262/not-recommended-to-write-out-use-strict-with-es6 correct

**Comment by fluffels** [comment]
@OOPMan thanks for the feedback, changes applied

**Comment by OOPMan** [comment]
@fluffels Okay, merging. Will tag a new release and push it out.

**Pull request merged**